numba/utils.py

```diff
@@ -287,6 +287,22 @@ def bit_length(intval):
         return len(bin(-intval - 1)) - 2
 
 
+def stream_list(lst):
```

Comment:
Le 04/01/2016 22:12, Siu Kwan Lam a écrit :
> In numba/utils.py
> <https://github.com/numba/numba/pull/1581#discussion_r48781746>:
> 
>> @@ -287,6 +287,22 @@ def bit_length(intval):
>>          return len(bin(-intval - 1)) - 2
>>  
>>  
>> +def stream_list(lst):
> 
> Can you explain why this is needed?

It makes it easy to implement RegistryLoader.  We want to be able to add
new implementations to a Registry that was already installed in a
context. We also don't want to add previously registered implementations
when "refreshing" the context. This helps us do that.